x-pack/plugins/lens/public/visualizations/datatable/components/table_basic.scss

```diff
@@ -1,5 +1,9 @@
 .lnsDataTableContainer {
   height: 100%;
+
```

Comment:
This is a good temporary solution to remove the color from the bottom of the table. However, it can cause some problems in the future if eui decided to change the implementation. It would be good to submit an issue in eui and ask the team what they advice. The options are: 
1. Keeping it as written here
2. Expose a background prop to datagrid (whether as a part of `gridStyle`)
3. Removing the background from eui (this code would not be needed)

Let's submit an issue to eui together tomorrow! (there are some gotchas to do so, for example using codesandbox)

Reply:
Created https://github.com/elastic/kibana/issues/193910